site/content/en/docs/Reference/uninstalling_minikube.md

```diff
@@ -0,0 +1,72 @@
+---

```

Comment:
I can't find it in the doc preview 
https://deploy-preview-5119--kubernetes-sigs-minikube.netlify.com/docs/reference/


how about adding the uninstal doc under each page separate, under where ppl look for install.

like here 
https://deploy-preview-5119--kubernetes-sigs-minikube.netlify.com/docs/start/windows/
and 
https://deploy-preview-5119--kubernetes-sigs-minikube.netlify.com/docs/start/macos/

Reply:
It's odd that it isn't in the preview - if I run `make site` locally from this PR, the page shows up. 

FWIW, I'm fine with this doc being in the reference section. I don't think this document is likely to be consulted often, but the folks who are looking for it are likely to be desperate for it.

I'm also not going to be terribly particular about what it looks like in this first iteration. I'm just appreciative that it's being done =)